articles/overview/understanding-quantum-computing.md

```diff
@@ -11,7 +11,7 @@ no-loc: ['Q#', '$$v']
 
 # Understanding quantum computing
 
-Quantum computing uses the principles of quantum mechanics to process information. Because of this, quantum computing requires a different approach than classical computing.  One example of this difference is the processor used in quantum computers.  Where classical computers use familiar silicon-based chips, quantum computers use quantum materials such as atoms, ions, photons, or electrons.  
+Quantum computing uses the principles of quantum mechanics to process information. Because of this, quantum computing requires a different approach than classical computing.  One example of this difference is the processor used in quantum computers.Where classical computers use familiar silicon-based chips, quantum computers use quantum systems such as atoms, ions, photons, or electrons. They use their quantum properties to represent bits that can be prepared in different quantum superpositions of 1 and 0.  
```

Comment:
```suggestion
Quantum computing uses the principles of quantum mechanics to process information. Because of this, quantum computing requires a different approach than classical computing.  One example of this difference is the processor used in quantum computers. Where classical computers use familiar silicon-based chips, quantum computers use quantum systems such as atoms, ions, photons, or electrons. They use their quantum properties to represent bits that can be prepared in different quantum superpositions of 1 and 0.  
```

Reply:
Thank you for pointing out this typo from my end, I have fixed it through commit [cd9d69c](https://github.com/MicrosoftDocs/quantum-docs-pr/pull/1042/commits/cd9d69ccdc32b8802cbd7051a38a32b31dc128a2). I also found a similar spacing typo in the edited version from my end fixed through commit [4cfc28a](https://github.com/MicrosoftDocs/quantum-docs-pr/pull/1042/commits/4cfc28a1b84f41165df51cbd311b1da0f53d98b4)